src/jobsautoreport/report.py

```diff
@@ -114,7 +120,13 @@ def _is_e2e_or_subsystem_class(job: JobDetails) -> bool:
         return "e2e" in job.name or "subsystem" in job.name
 
     def get_report(self, from_date: datetime, to_date: datetime) -> Report:
-        jobs = self._querier.query_all_jobs(from_date, to_date)
+        jobs = self._querier.query_jobs(from_date, to_date)
+        step_events = self._querier.query_step_events(from_date, to_date)
+        assisted_step_events = [
+            step_event
+            for step_event in step_events
+            if self._is_assisted_repository(step_event.job)
+        ]
```

Comment:
don't filter on assisted repo, we want all repos that may lease machines from equinix (... and we pay for them).